dateparser/languages/locale.py

```diff
@@ -184,10 +185,28 @@ def translate_search(self, search_string, settings=None):
             original_tokens, simplified_tokens = self._simplify_split_align(sentence, settings=settings)
             translated_chunk = []
             original_chunk = []
+            simplified_tokens_length = len(simplified_tokens)
+            skip_next_token = False
             for i, word in enumerate(simplified_tokens):
+                next_word = simplified_tokens[i + 1] if (simplified_tokens_length - 1) > i else ""
```

Comment:
I think putting the `i` first has a better semantic and it's easier to understand, because it's `i` which is changing.
```suggestion
                next_word = simplified_tokens[i + 1] if i < (simplified_tokens_length - 1) else ""
```

And applying what I mention above:

```suggestion
                next_word = simplified_tokens[i + 1] if i < last_token_index else ""
```

Reply:
Done 👍